test/cansocket.uts

```diff
@@ -0,0 +1,251 @@
+% Regression tests for the CANSocket
```

Comment:
These tests are wrong. They should not try to send packets to the network as it won't work on our build systems. Please either mock the tests, or implement dissection / build as done here https://github.com/secdev/scapy/blob/master/scapy/contrib/gtp.uts

Reply:
Are you sure of that ?
He managed to make the tests work with sudo enabled, and they look quite OK to me :/
He's performing real CAN socket tests, which shows that the implementation is correct

Reply:
they are all disabled in `test/config/`

Reply:
Only the Windows ones

Reply:
They won't work on Linux too.

Only testing the code by sending and receiving is a bad idea. Building/dissecting is way better, for example we can be sure that the packets can be read from PCAPs on all platforms.

Reply:
Got it